<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VisualTrainerWidget</class>
 <widget class="QWidget" name="VisualTrainerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="_pMainLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="_pMainSplitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="_pUpperSplitLayout">
       <item>
        <layout class="QHBoxLayout" name="_pTopLayout" stretch="0,0,1,0,0,0,2,0">
         <property name="spacing">
          <number>6</number>
         </property>
         <item>
          <widget class="QPushButton" name="_pShowLayerEditorButton">
           <property name="text">
            <string>Edit metrics...</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="_pToolButtonGroupLayout"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="_pZoomIn">
           <property name="toolTip">
            <string>Zoom in</string>
           </property>
           <property name="text">
            <string>+</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="_pZoomOut">
           <property name="toolTip">
            <string>Zoom out</string>
           </property>
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="_pFitToView">
           <property name="toolTip">
            <string>Fit to view</string>
           </property>
           <property name="text">
            <string>=</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="_pInfoButton">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Info</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="PiiScrollArea" name="_pSelectorScrollArea" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="PiiImageDisplay" name="_pImageDisplay" native="true"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PiiImageDisplay</class>
   <extends>QWidget</extends>
   <header>PiiImageDisplay.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PiiScrollArea</class>
   <extends>QWidget</extends>
   <header>PiiScrollArea.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
